v2/apiserver/internal/api/events_test.go

```diff
@@ -59,41 +55,18 @@ func TestEventsServiceCreate(t *testing.T) {
 		name       string
 		event      Event
 		service    EventsService
-		assertions func(EventList, error)
+		assertions func(meta.List[Event], error)
 	}{
 		{
 			name: "unauthorized",
 			service: &eventsService{
 				authorize: neverAuthorize,
 			},
-			assertions: func(_ EventList, err error) {
+			assertions: func(_ meta.List[Event], err error) {
 				require.Error(t, err)
 				require.IsType(t, &meta.ErrAuthorization{}, err)
 			},
 		},
-		{
-			name: "create single event for specified project; error getting " +
-				"project from store",
-			event: Event{
-				ProjectID: "blue-book",
-			},
-			service: &eventsService{
-				projectAuthorize: alwaysProjectAuthorize,
-				projectsStore: &mockProjectsStore{
-					GetFn: func(context.Context, string) (Project, error) {
-						return Project{}, errors.New("projects store error")
-					},
-					ListSubscribersFn: func(context.Context, Event) (ProjectList, error) {
-						return ProjectList{}, nil
-					},
-				},
-			},
-			assertions: func(_ EventList, err error) {
-				require.Error(t, err)
-				require.Contains(t, err.Error(), "error retrieving project")
-				require.Contains(t, err.Error(), "projects store error")
-			},
-		},
```

Comment:
This test case goes away after the change described [here](https://github.com/brigadecore/brigade/pull/1881/files#r841099739) because there's no longer a separate step for retrieving a specific project by ID.